nessus-agent/elx/init.sls

```diff
@@ -30,6 +30,75 @@ Print nessus-agent help:
         without linking the agent to the server.
 {%- endif %}
 
+{%- if (nessus.log_rotation_strategy) %}
+Add Log Rotation Strategy:
+  file.replace:
+    - name: '/opt/nessus_agent/var/nessus/log.json'
```

Comment:
Do you have a copy of what this file looks like by default, on Windows and Linux?

Reply:
The default `log.json` created after installation is the same on both Windows and Linux.

```json
{
    "reporters": [
        {
            "tags": [
                "response"
            ],
            "reporter": {
                "type": "file",
		"file": "/opt/nessus_agent/var/nessus/logs/www_server.log"
            },
            "format": "combined"
        },
        {
            "tags": [
                "log",
                "info",
                "warn",
                "error",
                "trace"
            ],
            "reporter": {
                "type": "file",
                "file": "/opt/nessus_agent/var/nessus/logs/backend.log"
            },
            "context": true,
            "format": "system"
        }
    ]
}
```